jerry-core/ecma/builtin-objects/ecma-builtins.c

```diff
@@ -180,16 +181,9 @@ ecma_builtin_is (ecma_object_t *obj_p, /**< pointer to an object */
   JERRY_ASSERT (obj_p != NULL && !ecma_is_lexical_environment (obj_p));
   JERRY_ASSERT (builtin_id < ECMA_BUILTIN_ID__COUNT);
 
-  if (JERRY_CONTEXT (ecma_builtin_objects)[builtin_id] == NULL)
-  {
-    /* If a built-in object is not instantiated,
-     * the specified object cannot be the built-in object */
-    return false;
-  }
-  else
-  {
-    return (obj_p == JERRY_CONTEXT (ecma_builtin_objects)[builtin_id]);
-  }
+  /* If a built-in object is not instantiated,
+   * the specified object cannot be the built-in object */
```

Comment:
I would change this comment:

If a built-in object is not instantiated, its value is NULL and that cannot be equal to a valid object.

Reply:
Good point! I've modified it.